OpenXR - Add Logitech Mx Ink Stylus Support

Kimau opened the pull request:
Added Interaction Profile support for the Logitech Stylus. 

In future work we may want to expose an `isStylus` or similar for XR Trackers, though that is likely outside the scope of this commit which is the core extension work.

https://github.com/user-attachments/assets/067f09f3-babb-4f38-9dc3-98804195ccfe


Vendor Ref: https://logitech.github.io/mxink/OpenXR.html

Kimau commented:
Pardon branch mess up, pushing new version in a bit

BastiaanOlij reviewed (approved):
LGTM, should squash the commits into a single commit first though.

AThousandShips commented:
Your PR contains unrelated changes from other commits, please rebase, see [here](https://docs.godotengine.org/en/latest/contributing/workflow/pr_workflow.html#the-interactive-rebase)

AThousandShips commented:
The author is also incorrect because of the previous squash, you can fix it with `git commit --amend --reset-author`

Kimau commented:
Thanks, I hate rebase always mess it up. Normally use the GH squash 🤦

Kimau committed:
OpenXR - Support for the Logitech MxInk Stylus

In future work we may want to expose an isStylus or similar for
XR Trackers, though that is likely outside the scope of this commit
which is the core extension work.

Vendor Ref: https://logitech.github.io/mxink/OpenXR.html

akien-mga commented:
Thanks!
